src/repositories/xref-repository.ts

```diff
@@ -295,32 +300,44 @@ export class XrefRepository extends Repository {
    *
    * @async
    * @param {IMeasurementSearch} search - Search properties.
+   * @param {number[]} tsns - filters for measurements applied to any of the specified TSNs
    * @returns {Promise<ITsnQualitativeMeasurement[]>}
    */
-  async searchForQualitativeMeasurements(search: IMeasurementSearch): Promise<ITsnQualitativeMeasurement[]> {
+  async searchForQualitativeMeasurements(
+    search: IMeasurementSearch,
```

Comment:
We should move the tsns into the search object

Reply:
Done